cmd/minishift/cmd/start.go

```diff
@@ -384,8 +384,8 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 		RemoteSSHUser:         viper.GetString(configCmd.RemoteSSHUser.Name),
 		SSHKeyToConnectRemote: viper.GetString(configCmd.SSHKeyToConnectRemote.Name),
 	}
-	minishiftConfig.InstanceConfig.VMDriver = machineConfig.VMDriver
-	minishiftConfig.InstanceConfig.Write()
+	minishiftConfig.InstanceStateConfig.VMDriver = machineConfig.VMDriver
```

Comment:
This is a tricky one to call 'state', maybe from our perspective it is... right...